modules/custom/d_commerce/d_commerce.install

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * @file
+ * Install, update, and uninstall functions for d_commerce.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function d_commerce_install() {
+  \Drupal::service('d_content_init.config_update')
+    ->updateThemeInConfigs('d_commerce', '/config/optional', 'block.block');
+}
```

Comment:
@tomasz-makiel-droptica tu nie ma unistall umślnie? bo user mógł sobie coś z tymi blokami zrobić? jak dla mnie ma to sens, samego commerca user musialby odinstalowac zeby te bloki przestaly dzialac ..

Reply:
No tak też pomyślałem dlatego nie uswam tych konfiguracji